nl/mail/chrome/messenger/safeMode.dtd

```diff
@@ -36,33 +36,32 @@
 #endif
    - ***** END LICENSE BLOCK ***** -->
 
-<!ENTITY changeAndRestartButton2.label     "Save and Restart">
+<!ENTITY changeAndRestartButton2.label     "Opslaan en Herstarten">
 <!ENTITY changeAndRestartButton2.accesskey "S">
 
 
-<!ENTITY safeModeOptionsWindow.width       "width: 49em;">
+<!ENTITY safeModeOptionsWindow.width       "breedte: 49em;">
 <!ENTITY safeModeOptionsDialog.title       "&brandShortName; Trouble Shooting">
 
-<!ENTITY safeModeOptionsDescription2.label "If you are experiencing difficulty with Postbox, try restarting 
-with one or more of these trouble shooting options selected:">
+<!ENTITY safeModeOptionsDescription2.label "Als u problemen verwacht met Postbox, probeer dan om te herstarten met een of meerdere trouble shooting opties geselecteerd:">
```

Comment:
CORRECTION NEEDED, it should be: "Als u problemen **ervaart** met Postbox, selecteer een of meerdere van deze trouble shooting opties en probeer te herstarten:''